DigitalRain/Raindrops/StandardRaindrop.cs

```diff
@@ -5,91 +5,42 @@
 namespace DigitalRain.Raindrops
 {
     using Columns;
-    using GameUtilities;
-    public class StandardRaindrop : IRaindrop, IGameObject
+
+    public class StandardRaindrop : IRaindrop
     {
-        // IRaindrop
-        public char[] SymbolPool { get; private set; }
-        public char Symbol
-        {
-            get { return _symbol; }
-            set {_symbol = value; }
-        }
+        private readonly ColumnSpace _columnSpace;
+        private readonly char _symbol;
         public double LifeRemaining { get; private set; }
-        public static Color DefaultColor { get { return Color.GreenYellow; } }
```

Comment:
So glad you got rid of this DefaultColor garbage! What was I thinking?! You should not be able to define a `ColorCalculator` without specifying a color.